README.md

```diff
@@ -383,6 +385,9 @@ You will also need to ensure that Krun is restarted once the machine has
 rebooted. The `etc/` directory contains example `/etc/rc.local` files for the
 platforms supported by Krun.
 
+You should turn off "turbo boost" in your BIOS if you can. On Linux Krun will
```

Comment:
Why tell people to turn it off if Krun does it anyway? Why not say "Krun will disable turbo mode when it runs (or complain if it is unable)."?

Reply:
I'd have more confidence is a system which reports it is incapable of turbo mode (which is what you get if you disable in the bios).

Reply:
Tthe later APERF/MPERF check acts as a guarantee that, even were our turbo-mode-off thing fail, we'd still catch the problem. So I think we should just kill this sentence entirely.